docs/source/quickstart.rst

```diff
@@ -6,24 +6,37 @@ Quickstart
 Installing MLflow
 -----------------
 
-You install MLflow by running:
+You install MLflow with extra dependencies by running:
 
-.. code-section::
+.. plain-section::
+  
+  .. container:: python
   
     .. code-block:: python
 
-        pip install mlflow
+      pip install mlflow[extras]
+      
+    where the extra dependencies are:
 
-    .. code-block:: R
+    - ``scikit-learn`` when Python version >= '3.5'
+    - ``scikit-learn`` == 0.20 when Python version < '3.5'
+    - ``boto3`` >= 1.7.12
+    - ``mleap`` >= 0.8.1
+    - ``azure-storage``
+    - ``google-cloud-storage``
```

Comment:
There's no need to list these in the Quickstart. The Quickstart should be a very simple, fast way to get started.

Reply:
Ditto ^.

Reply:
@mateiz @stbof The quickstart is currently the only section of our docs that discusses how to install MLflow. Perhaps we should introduce an "Installing MLflow" section with info about how to install MLflow from pip (with/without extras), from maven for Java, and from CRAN for R.

Reply:
Makes sense to me. Pretty standard doc section too.